examples/full-s3-repo/index.js

```diff
@@ -38,9 +38,15 @@ const repo = new Repo('/tmp/test/.ipfs', {
 
 // Create a new IPFS node with our S3 backed Repo
 let node = new IPFS({
-  repo
+  repo,
+  config: {
+    Discovery: { MDNS: { Enabled: false }, webRTCStar: { Enabled: false } },
+    Bootstrap: []
+  }
 })
 
+console.log('Start the node')
```

Comment:
Please remove this `log`

Reply:
@vasco-santos this log is part of the example and is meant to be informational to the user.